packages/simulator/web/routes.js

```diff
@@ -127,10 +125,10 @@ function register(io) {
           capacity,
         })
       ),
-      bufferTime(100)
+      throttleTime(50)
```

Comment:
This line is changed to only visualize the latest bus update... Does changing this cause some side effects to the bookings?